index.html

```diff
@@ -14,47 +14,26 @@
 
     <link rel="stylesheet" href="style.css">
 
+    <!-- Favicon courtesy of Twemoji at https://twemoji.twitter.com -->
     <link rel="icon" href="assets/images/favicon.ico" type="image/x-icon">
 </head>
+    <body>
+        <main class="center">
+            <h1 class="vertical-offset padding-bottom">Hi, I'm Reece <span class="animated-wave emoji">👋</span></h1>
 
-<body>
-    <main class="center">
-        <section class="flex-row">
-            <h1 class="vertical-offset">Hi, I'm Reece</h1>
-            <h1 class="animated-wave">👋</h1>
-        </section>
+            <h2 class="padding-bottom">Software Engineer, Flutter Fan and All Around Great Person</h3>
 
-        <div aria-hidden="true" role="separator" class="spacer"></div>
-
-        <h3>Software Engineer, Flutter Fan and All Around Great Person</h3>
-
-        <div aria-hidden="true" role="separator" class="spacer"></div>
-
-        <section class="inner-div">
-            <a href="https://github.com">
-                <section class="flex-row">
+            <div class="contact-details">
+                <h3 class="sr-only">Contact info</h3>
+                <a class="contact-row" href="https://github.com">
```

Comment:
🟡 It's nitpicky, but `contact-row` isn't really semantically appropriate. This is an item in a row of contact items. Why not replace `contact-details` with `contact-row` & `contact-row` with `contact-item`?

Or better yet, with `row` and `row-item` specifically? This CSS could (*in a hypothetical larger project) be reused